flutter/shell/platform/tizen/BUILD.gn

```
@@ -175,6 +175,7 @@ template("embedder") {
     if (api_version == "6.5" && target_name != "flutter_tizen_wearable") {
```

Do we still need this?
```suggestion
    if (api_version == "6.5") {
```
I don't quite remember the exact reason why the wearable profile was excluded (binary size? non-public symbol issue?).

---

I remember that .net(NUI) is not supported in wearable.
In this case, some APIs mentioned in the PR message do not exist in wearable header.
This part can be removed later by deprecating the wearable.